<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyNotePad</class>
 <widget class="QMainWindow" name="MyNotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyNotePad</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/** widgets settings **/

#centralWidget
{
background-color: rgb(255, 170, 121);
}

#widget
{
border-radius: 10px;
border-image: url(&quot;:/Icon/Images/28_gradient_background.jpg&quot;);
}

/**  button settings **/

#btn_text_editor
{
border-image: url(&quot;:/Icon/Images/16_Text_icon.png&quot;);
margin:10px;
}

#btn_text_editor:hover
{
border-image: url(&quot;:/Icon/Images/16_Text_icon.png&quot;);
margin:0px;
}

#btn_image_converter
{
border-image:url(&quot;:/Icon/Images/18_PDF_Documents_icon.png&quot;);
margin:10px;
}

#btn_image_converter:hover
{
border-image:url(&quot;:/Icon/Images/18_PDF_Documents_icon.png&quot;);
margin:0px;
}

#btn_open
{
border-image:url(&quot;:/Icon/Images/20_folders_open_icon.png&quot;);
margin:10px;
}

#btn_open:hover
{
border-image:url(&quot;:/Icon/Images/20_folders_open_icon.png&quot;);
margin:0px;
}

#btn_save_as_pdf
{
border-image:url(&quot;:/Icon/Images/21_pdf_icon.png&quot;);
margin:10px;
}

#btn_save_as_pdf:hover
{
border-image:url(&quot;:/Icon/Images/21_pdf_icon.png&quot;);
margin:0px;
}

#btn_remove_image
{
border-image:url(&quot;:/Icon/Images/25_edit_clear_icon.png&quot;);
margin:10px;
}

#btn_remove_image:hover
{
border-image:url(&quot;:/Icon/Images/25_edit_clear_icon.png&quot;);
margin:0px;
}

#btn_back
{
border-image:url(&quot;:/Icon/Images/23_Home_icon.png&quot;);
margin:10px;
}

#btn_back:hover
{
border-image:url(&quot;:/Icon/Images/23_Home_icon.png&quot;);
margin:0px;
}
/** label settings **/

#lbl_image_preview
{
background-image: url(&quot;:/Icon/Images/24_gradient_image.jpg&quot;);
border: 4px solid rgb(0, 0, 127 );
border-radius: 10px;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_main">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <widget class="QPushButton" name="btn_text_editor">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>415</y>
             <width>300</width>
             <height>300</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>15</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_image_converter">
           <property name="geometry">
            <rect>
             <x>1130</x>
             <y>420</y>
             <width>300</width>
             <height>300</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_title">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>170</y>
             <width>251</width>
             <height>121</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>35</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Text Editor</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_shortcut_texteditor">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>830</y>
             <width>121</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Rod</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Shortcut key:
  CTRL + T</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_shortcut_imagetoPDF">
           <property name="geometry">
            <rect>
             <x>1220</x>
             <y>820</y>
             <width>131</width>
             <height>71</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Rod</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Shortcut key
  CTRL + O</string>
           </property>
          </widget>
          <widget class="QLabel" name="lbl_title_2">
           <property name="geometry">
            <rect>
             <x>1040</x>
             <y>160</y>
             <width>501</width>
             <height>131</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>35</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Image to PDF Converter</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_notepad">
       <widget class="QTextEdit" name="txt_pad">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1901</width>
          <height>1001</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_image">
       <widget class="QLabel" name="lbl_image_preview">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>1861</width>
          <height>881</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image Preview</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_save_as_pdf">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>960</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_open">
        <property name="geometry">
         <rect>
          <x>780</x>
          <y>960</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>90</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_back">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>960</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_remove_image">
        <property name="geometry">
         <rect>
          <x>1330</x>
          <y>960</y>
          <width>90</width>
          <height>90</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_open">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>920</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_print">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>920</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Print PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_remove">
        <property name="geometry">
         <rect>
          <x>1310</x>
          <y>920</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Remove Image</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_home">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>920</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_as_PDF"/>
    <addaction name="actionBack_to_Index"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack_to_Index"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionPrint_as_PDF"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/1_New_icon.png</normaloff>:/Icon/Images/1_New_icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/2_Open_icon.png</normaloff>
     <normalon>Images/2_Open_icon.png</normalon>:/Icon/Images/2_Open_icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/3_save_icon.png</normaloff>:/Icon/Images/3_save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/4_saveas_icon.png</normaloff>:/Icon/Images/4_saveas_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/8_Cut_icon.png</normaloff>:/Icon/Images/8_Cut_icon.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/7_copy_icon.png</normaloff>:/Icon/Images/7_copy_icon.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/9_Paste_icon.png</normaloff>:/Icon/Images/9_Paste_icon.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionBack_to_Index">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/11_Home_icon.png</normaloff>:/Icon/Images/11_Home_icon.png</iconset>
   </property>
   <property name="text">
    <string>Back to Index</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/6_Undo_icon.png</normaloff>:/Icon/Images/6_Undo_icon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/5_Redo_icon.png</normaloff>:/Icon/Images/5_Redo_icon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionPrint_as_PDF">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icon/Images/10_PDF_icon.png</normaloff>:/Icon/Images/10_PDF_icon.png</iconset>
   </property>
   <property name="text">
    <string>Print as PDF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
